ehitchhiking-ui/src/app/shared/components/menu/menu.component.ts

```diff
@@ -22,7 +22,9 @@ export class MenuComponent implements OnInit {
 	}
 	openBlacklistDialog(): void {
 		const dialogRef = this.dialog.open(BlacklistComponent, {
-			width: '400px',
+			width: '23rem',
+			panelClass: 'mat-dialog-no-padding',
```

Comment:
pls create a constant for 'mat-dialog-no-padding'

Reply:
fixed